app/modules/import-export/runners/export/site-settings.php

```diff
@@ -69,4 +75,30 @@ public function export_theme() {
 
 		return $theme_data;
 	}
+
+	private function export_experiments() {
+		$features = Plugin::$instance->experiments->get_features();
+
+		if ( empty( $features ) ) {
+			return null;
+		}
+
+		$experiments_data = [];
+
+		foreach ( $features as $feature_name => $feature ) {
+			if ( ! $feature['mutable'] ) {
+				continue;
+			}
```

Comment:
Just want to double check if this condition is correct. Not sure what mutable means.